src/late_multicellular_stage/MulticellularStage.cs

```diff
@@ -635,6 +637,23 @@ public void OnSocietyFounded(PlacedStructure societyCenter)
         animationCamera.ReParent(rootOfDynamicallySpawned);
     }
 
+    public void UpdateBackgroundPanorama()
+    {
+        if (CurrentGame != null)
+        {
+            if (CurrentGame.GameWorld.Map.CurrentPatch != null)
```

Comment:
You can combine this and the previous if with null propagation (this is what the CI is complaining about).